src/gui/MainWindow.cpp

```diff
@@ -370,6 +370,7 @@ MainWindow::MainWindow()
 
 #ifdef Q_OS_MACOS
     setUnifiedTitleAndToolBarOnMac(true);
+    this->setStyleSheet("QToolButton {color:white;}");
```

Comment:
You cannot do this. It may be working for your specific dark mode setting, but it'll be unreadable in classic light mode. Generally, do not hard-code colours. It will break things.

Reply:
Now uses MacUtils DarkMode detection and only then adds stylesheet